### DISCUSSION TITLE
ExpressVPN

### DISCUSSION THREAD

**Discussion by giscus** [AwaitQuality/windowsonarm#26] [General]
# ExpressVPN

Check if ExpressVPN is ARM ready on Windows and can be used on Windows ARM devices such as the Surface with the Snapdragon X Elite.

https://windowsonarm.org/b2d6ef1b-71bd-4853-9885-1159accb683a

**Comment by talynone** [comment]
Native client in testing, as a work-around you can download the ARM version of OpenVPN here:
https://openvpn.net/community-downloads/

Then use an OpenVPN configuration from here: 
https://www.expressvpn.com/support/vpn-setup/manual-config-for-windows-xp-vista-7-8-with-openvpn/

**Comment by talynone** [comment]
Beta download for CoPilot PC's now available:

https://www.expressvpn.com/vpn-download/vpn-windows

**Reply by OpenSource03** [reply]
I've updated the status. Note that status change may not reflect momentarily due to 2h edge caching.
